test/e2e/queue.go

```diff
@@ -461,15 +463,31 @@ var _ = Describe("AppWrapper E2E Test", func() {
 		Expect(err).NotTo(HaveOccurred(), "Expecting pods for app wrapper: aw-ff-deployment-2-340-cpu")
 		fmt.Fprintf(GinkgoWriter, "[e2e] MCAD Scheduling Fail Fast Preemption Test - Pods not found for app wrapper aw-ff-deployment-2-340-cpu\n")
 
-		// Make sure aw2 pods do not exist
 		err = waitAWPodsReady(context, aw3)
 		Expect(err).NotTo(HaveOccurred(), "Expecting no pods for app wrapper: aw-ff-deployment-2-340-cpu")
 		fmt.Fprintf(GinkgoWriter, "[e2e] MCAD Scheduling Fail Fast Preemption Test - Ready pods found for app wrapper aw-ff-deployment-2-340-cpu\n")
 
-		// Make sure pods from AW aw-deployment-1-850-cpu above do not exist proving preemption
-		time.Sleep(5 * time.Minute)
-		err = waitAWAnyPodsExists(context, aw2)
-		Expect(err).To(HaveOccurred(), "Expecting no pods for app wrapper : aw-ff-deployment-1-850-cpu")
+		// Make sure pods from AW aw-deployment-1-850-cpu have preempted
+		var pass = false
```

Comment:
why not use the `err := wait.Poll(....)` method? There is an example on how to do it in the 50 generic pods test bellow. I believe the code is easier to read, comprehend.

Reply:
Thanks, I think wait. poll() waits for pods. I want to know the AW status. I propose we stop checking pods and move all our tests to check AW status since typically users interact with AWs.

Reply:
I created issue for it here: https://github.com/project-codeflare/multi-cluster-app-dispatcher/issues/520